map/manage-bookmarks/src/main/java/com/esri/samples/manage_bookmarks/ManageBookmarksSample.java

```diff
@@ -117,9 +122,10 @@ public void start(Stage stage) {
       // add label and bookmarks to the control panel
       controlsVBox.getChildren().addAll(bookmarkLabel, bookmarkNames, addBookmarkButton);
 
-      ArcGISMap map = new ArcGISMap(Basemap.createImageryWithLabels());
+      // create a map with the imagery standard basemap style
```

Comment:
Same comment as in the other PR, might be worth checking if the plain english description is `standard imagery basemap`, as opposed to the technical enum name?

Reply:
I'm just thinking on most of them I think it just says "with a basemap style". Maybe that is easier. Any thoughts @Rachael-E ?